pyproject.toml

```diff
@@ -29,6 +29,7 @@ markers = ["platform_specific_behavior"]
 # coverage will gather stats for it, but it doesn't correspond to a real source file,
 # so reporting will fail, unless we omit it here.
 omit = ["src/python/pants/__init__.py"]
+include_namespace_packages = true
```

Comment:
It only passes with this line. I tried to render a `pyproject.toml` like you mention in your other comment, but didn't get that. This is the file I got the contents from in the `get_namespace_value_from_config` function I added.

It really feels like a hack, but I'm not sure how to otherwise get the value.

Reply:
Figured out how to use the generated file, I didn't use include `{tmpdir}` in the `--coverage-py-config` path earlier.